### ISSUE TITLE
Interactive UI Elements?

### ISSUE THREAD

**Issue by david-littlefield** [gildas-lormeau/SingleFile#980]
Is there a way for SingleFile to keep the event listeners for UI elements? 

For example, clicking on the menu icon to display a dropdown menu.

**Comment by gildas-lormeau** [comment]
Yes and no. Most of the time, it won't work because scripts are not designed to be saved for offline reading. You can find more info here: https://github.com/gildas-lormeau/SingleFile/blob/master/faq.md#why-dont-interactive-elements-like-folding-titles-dynamic-maps-or-carousels-work-properly-in-saved-pages

**Comment by david-littlefield** [comment]
Gotcha. That would save all scripts, not just user interface elements, correct?

**Comment by gildas-lormeau** [comment]
Yes, all the scripts would be saved.
